packages/nimble-components/src/rich-text-editor/specs/README.md

```diff
@@ -320,20 +335,69 @@ markdown based on [CommonMark](http://commonmark.org/) flavor:
 -   Italics - `*Italics*`
 -   Numbered list - `1. Numbered list`
 -   Bulleted list - `* Bulleted list`
+-   Absolute URL links - `<Absolute URI link>` (For more details on the markdown syntax for absolute URL links, see [Autolinks in CommonMark](https://spec.commonmark.org/0.30/#autolink))
+
+_Configurations on Tiptap to support only absolute links_:
+
+Install the [link extension](https://tiptap.dev/api/marks/link) mark from Tiptap and initialize the `Links` with the following configurations:
+
+1.  Set regular expression in [validate](https://tiptap.dev/api/marks/link#validate) field to support only `HTTP` and `HTTPS` absolute links in the editor.
+2.  Set [openOnClick](https://tiptap.dev/api/marks/link#open-on-click) to `false` for the editor, to restrict the user from opening a link from the editor by clicking. Users can open the link only by
+    `Right-click >> Open link in new tab` from the editor.
```

Comment:
Does this mean that link markdown in the editor is somehow rendered as an operable link? I would have expected it to just look like plain markdown text, e.g. "<http://ni.com>". Only in the viewer would I expect it to look and behave like a link.

Reply:
The rich text editor is a wysiwyg editor. The user does not type markdown into it so they would not see the commonmark markdown representation, they would see it render as a link.

It will render as a link during edit (in WYSIWYG fashion) but not be operable as to avoid accidental navigation (except by new window interactions). I agree that it's not a perfect tradeoff and they are not clear how to operate. But I think it's okay enough for now and the UX could be iterated on as needed.

FWIW powerpoint as a WYSIWYG editor also prevents link clicks from opening by default and they include in the alt text instructions for how to follow the link:

![image](https://github.com/ni/nimble/assets/1588923/621eec1a-d3fe-4dfe-a1b6-564ba3b3b94e)

That's just one example though, finding the right UX may take additional investigation.